### PR TITLE
Update page layout

### PR DESCRIPTION
**Pull request by mrcmatuszak**

### CODE REVIEW

File: frontend/src/main/scala/jess/GameBoardApp.scala [hunk 1]
```diff
@@ -1,24 +1,88 @@
...
+    }
+
   }
 
-  def appendPar(targetNode: dom.Node, text: String): Unit = {
-    val parNode = document.createElement("p")
-    val textNode = document.createTextNode(text)
-    parNode.appendChild(textNode)
-    targetNode.appendChild(parNode)
+  private def joinChat(ws: String): Unit = {
```
Comment (lukeindykiewicz) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: joinChat?

File: frontend/src/main/scala/jess/GameBoardApp.scala [hunk 2]
```diff
@@ -1,24 +1,88 @@
...
+        )
+      )
+    ).render
+  }
+
+
+  private def eventList(events: Seq[GameEvent]) = {
+    ul(
+      `class` := "list-group",
+      css.maxHeight := 200.px,
```
Comment (lukeindykiewicz) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: Wouldn't it be better to move css to separate file?

File: frontend/src/main/scala/jess/GameBoardApp.scala [hunk 3]
```diff
@@ -1,24 +1,88 @@
...
+  val joinButton = dom.document.getElementById("ws-join").asInstanceOf[HTMLButtonElement]
+  val wsAddressInput = dom.document.getElementById("ws-address").asInstanceOf[HTMLInputElement]
+  val rnd = util.Random
+  val ev = scala.collection.mutable.ArrayBuffer.empty[GameEvent]
 
   @JSExport
   def main(): Unit = {
-    appendPar(document.body, "Jess Game Board")
+
+    wsAddressInput.placeholder = "127.0.0.1:8090/stream"
```
Comment (lukeindykiewicz) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: Move server address to some config.

Comment (mrcmatuszak) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: I put it there that we don't have to type it all the time. I removed it.

File: frontend/src/main/scala/jess/GameBoardApp.scala (hunk 2 above)
Comment (mrcmatuszak) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: ok

File: frontend/src/main/scala/jess/GameBoardApp.scala (hunk 1 above)
Comment (mrcmatuszak) [inline frontend/src/main/scala/jess/GameBoardApp.scala]: :D renamed

**Pull request merged**